HealthSciencesThemePlugin.inc.php

```diff
@@ -33,10 +33,8 @@ public function init() {
 		$additionalLessVariables = [];
 		if ($this->getOption('baseColour') !== '#10BECA') {
 			$additionalLessVariables[] = '@primary:' . $this->getOption('baseColour') . ';';
-			if (!$this->isColourDark($this->getOption('baseColour'))) {
```

Comment:
If user chooses a white-ish base color, buttons and a text do not have enough contrast. Probably, in this case, we can add a border for buttons and make the text darker. So, I would rather leave this check.

Reply:
Ah, I have a simpler solution -- I need to keep these three variables but I will just add a new "contrasting colour" variable, as with the Pragma theme.